test/uk/gov/hmrc/emcstfe/models/response/GetMovementResponseSpec.scala

```diff
@@ -903,143 +903,143 @@ class GetMovementResponseSpec extends UnitSpec with GetMovementFixture {
-            |             <body:Package>
-            |                <body:KindOfPackages>BO</body:KindOfPackages>
-            |                <body:NumberOfPackages>125</body:NumberOfPackages>
-            |                <body:CommercialSealIdentification>SEAL456789321</body:CommercialSealIdentification>
-            |                <body:SealInformation language="en">Red Strip</body:SealInformation>
-            |              </body:Package>
-            |						</body:BodyEadEsad>
-            |						<body:BodyEadEsad>
-            |							<body:BodyRecordUniqueReference>2</body:BodyRecordUniqueReference>
-            |							<body:ExciseProductCode>W300</body:ExciseProductCode>
-            |							<body:CnCode>27111901</body:CnCode>
-            |							<body:Quantity>501</body:Quantity>
-            |							<body:GrossMass>901</body:GrossMass>
-            |							<body:NetMass>475</body:NetMass>
-            |             <body:AlcoholicStrengthByVolumeInPercentage>12.7</body:AlcoholicStrengthByVolumeInPercentage>
-            |							<body:FiscalMark language="en">FM564790 Fiscal Mark</body:FiscalMark>
-            |							<body:FiscalMarkUsedFlag>1</body:FiscalMarkUsedFlag>
-            |							<body:DesignationOfOrigin language="en">Designation of Origin</body:DesignationOfOrigin>
-            |							<body:SizeOfProducer>20000</body:SizeOfProducer>
-            |							<body:CommercialDescription language="en">Retsina</body:CommercialDescription>
-            |							<body:BrandNameOfProducts language="en">BrandName</body:BrandNameOfProducts>
-            |             <body:Package>
-            |                <body:KindOfPackages>BO</body:KindOfPackages>
-            |                <body:NumberOfPackages>125</body:NumberOfPackages>
-            |                <body:CommercialSealIdentification>SEAL456789321</body:CommercialSealIdentification>
-            |                <body:SealInformation language="en">Red Strip</body:SealInformation>
-            |              </body:Package>
-            |              <body:Package>
-            |                <body:KindOfPackages>HG</body:KindOfPackages>
-            |                <body:NumberOfPackages>7</body:NumberOfPackages>
-            |                <body:CommercialSealIdentification>SEAL77</body:CommercialSealIdentification>
-            |                <body:SealInformation language="en">Cork</body:SealInformation>
-            |              </body:Package>
-            |						</body:BodyEadEsad>
-            |					</body:EADESADContainer>
-            |				</body:Body>
-            |			</body:IE801>
-            |		</mov:currentMovement>
-            |	</mov:movementView>""".stripMargin)
-
-        GetMovementResponse.xmlReader.read(noConsignorXML) shouldBe ParseFailure(EmptyError(GetMovementResponse.consignorName))
-      }
+//      "missing consignorName" in {
```

Comment:
Are these tests no possible now? Or do they still need to be updated?

Reply:
Updated test and pushed another commit